wiremock/deployment/src/main/java/io/quarkiverse/openapi/wiremock/generator/deployment/wiremock/OpenApi2WiremockMapper.java

```diff
@@ -0,0 +1,164 @@
+package io.quarkiverse.openapi.wiremock.generator.deployment.wiremock;
+
+import static io.swagger.v3.parser.util.SchemaTypeUtil.*;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+
+import io.quarkiverse.openapi.wiremock.generator.deployment.wiremock.model.Request;
+import io.quarkiverse.openapi.wiremock.generator.deployment.wiremock.model.Response;
+import io.quarkiverse.openapi.wiremock.generator.deployment.wiremock.model.Stubbing;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.Operation;
+import io.swagger.v3.oas.models.PathItem;
+import io.swagger.v3.oas.models.Paths;
+import io.swagger.v3.oas.models.media.MediaType;
+import io.swagger.v3.oas.models.media.Schema;
+import io.swagger.v3.oas.models.responses.ApiResponse;
+
+public class OpenApi2WiremockMapper {
+
+    private final OpenAPI openAPI;
+    public static final String ONLY_SUPPORTED_MEDIA_TYPE = "application/json";
+    public static final Integer DEFAULT_STATUS_CODE = 200;
+
+    public OpenApi2WiremockMapper(OpenAPI openAPI) {
+        this.openAPI = openAPI;
+    }
+
+    public List<Stubbing> generateWiremockStubbings() {
+
+        List<Stubbing> stubbings = new ArrayList<>();
+
+        Paths paths = openAPI.getPaths();
+
+        Set<Map.Entry<String, PathItem>> pathsEntries = paths.entrySet();
+
+        for (Map.Entry<String, PathItem> pathEntry : pathsEntries) {
+            stubbings.addAll(generateStubbings(pathEntry));
+        }
+
+        return stubbings;
+    }
+
+    private List<Stubbing> generateStubbings(Map.Entry<String, PathItem> entry) {
+        Set<Stubbing> stubbings = new HashSet<>();
+
+        String pathName = entry.getKey();
+
+        PathItem pathItem = entry.getValue();
+
+        Map<PathItem.HttpMethod, Operation> operationsMap = pathItem.readOperationsMap();
+
+        operationsMap.forEach(
+                (httpMethod, operation) -> {
+                    Stubbing.StubbingBuilder builder = new Stubbing.StubbingBuilder();
+
+                    Request request = Request.create(pathName, httpMethod.name());
+
+                    int statusCode = getStatusCode(operation);
+
+                    Response response = Response.create(
+                            statusCode,
+                            getResponseBody(String.valueOf(statusCode), operation));
+
+                    Stubbing stubbing = builder
+                            .request(request)
+                            .response(response)
+                            .build();
+
+                    stubbings.add(stubbing);
+                });
+
+        return stubbings.stream().toList();
+    }
+
+    private String getResponseBodyFrom$Ref(String component) {
```

Comment:
No Helber, I wanted to follow the OpenAPI specification name, I can change here.

Reply:
The compiler uses `$` to name inner classes. I'm not sure it would cause issues in real life, but I'd avoid it.
It may also confuse log readers as well. They may think that `Ref` is an inner class.

Reply:
It makes total sense!